src/herder/TxQueueLimiter.cpp

```diff
@@ -117,22 +120,29 @@ TxQueueLimiter::canAddTx(TransactionFrameBasePtr& newTx,
     {
         if (feeRate3WayCompare(*it, newTx) >= 0)
         {
-            return false;
+            int64 minFee;
+            // compute a fee strictly better than `it`
+            // this won't overflow because fee bids for valid transactions are
```

Comment:
This comment is wrong. This calculation happens _before_ the available balance check, and fee-bump transactions have `int64` fees. So this can overflow, and you need to be more careful.

Reply:
updated the code to not care if we overflow